src/Corporation/ui/Helpers.tsx

```diff
@@ -4,7 +4,7 @@ import { IIndustry } from "../IIndustry";
 // current industry.
 export function isRelevantMaterial(matName: string, division: IIndustry): boolean {
   // Materials that affect Production multiplier
-  const prodMultiplierMats = ["Hardware", "Robots", "AICores", "RealEstate"];
+  const prodMultiplierMats = ["Hardware", "Robots", "AICores", "RealEstate", "AI Cores", "Real Estate"];
```

Comment:
hmm why was this needed? shouldn't we be standardizing the mats somewhere already?

Reply:
Yes, it's subject to refactor some code, i did only the quick bugfix

Reply:
sorry i mean Ai Cores and Rel Estate are already in the array? but you have added them again?